src/util/comparison.coffee

```diff
@@ -45,6 +45,8 @@ module.exports.greaterThanOrEquals = (a, b, precision = DateTime.Unit.MILLISECON
 module.exports.equivalent = equivalent = (a, b) ->
   if a == null && b == null
     return true
+  else if a == null || b == null
```

Comment:
`a == null` will get transpiled to `a === null` in JavaScript, which will fail for `undefined` input.  What we really want is `a == null` in Javascript which is `a?` in CoffeeScript.

Lines 46-49 on this branch should go away when the merge conflict with https://github.com/cqframework/cql-execution/pull/9 gets resolved.  It might be best to replace them here just to reduce the risk that these lines make it through instead somehow.

They should instead be:
```
return true if not a? and not b?
return false unless a? and b?
```